src/components/Crypto/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React, {FC} from 'react';
+import {CryptoTypes} from '../../types';
+import ArrowDown from '../../assets/ArrowDown.png';
+import ArrowUp from '../../assets/ArrowUp.png';
+import {
+  CryptoContainer,
+  NameContainer,
+  Price,
+  Logo,
+  Name,
+  Slug,
+  Arrow,
+  ArrowPercentage,
+  Percentage,
+} from '../Crypto/styles';
+
+const Crypto: FC = ({item}: {item: CryptoTypes}) => (
+  <CryptoContainer>
+    <Logo source={item.image} />
+    <NameContainer>
+      <Name>{item.name}</Name>
+      <Slug>{item.slug}</Slug>
+    </NameContainer>
+    <Price>${item.price}</Price>
+    {parseInt(item.market_porcentage, 10) > 0 ? (
+      <Percentage Boolean={true}>{item.market_porcentage}%</Percentage>
```

Comment:
Use  a more descriptibe name for the prop, 
also put the validacion in the prop:

```
 <Percentage Boolean={parseInt(item.market_porcentage, 10) > 0 }>{item.market_porcentage}%</Percentage>
```